src/components/Tasks/TaskItem.jsx

```diff
@@ -0,0 +1,20 @@
+import { formatDate } from "@utils/formatDate.js";
+import PropTypes from "prop-types";
+import "./style.scss";
+
+function TaskItem({ task = {} }) {
```

Comment:
Why task = {} ?

Reply:
I’m using it as a default value for this `task` property. I think it is a good practice to add a default so that the app will never through an error while destructuring the property. I can even write `task || {}` to avoid crashing the app.
Please suggest to me the best practice here.
Thank you.

Reply:
As we discussed, You can use `defaultProps` here for default values.